### PR TITLE
[bitnami/nginx] Chart standardised

### PR DESCRIPTION
**Pull request by migruiz4**
**Description of the change**

This PR standardizes the bitnami/nginx chart to be in line with the rest of the catalog.

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the values.yaml and added to the README.md using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/nginx/templates/NOTES.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ APP VERSION: {{ .Chart.AppVersion }}
 
 NGINX can be accessed through the following DNS name from within your cluster:
 
-    {{ include "common.names.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }} (port {{ .Values.service.port }})
+    {{ include "common.names.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }} (port {{ .Values.service.ports.http }})
```
Comment (fmulero) [inline bitnami/nginx/templates/NOTES.txt:9]: please add in README.md file information about `diagnosticMode`:

https://github.com/bitnami/charts/blob/master/template/CHART_NAME/templates/NOTES.txt#L7-L25

**Review by fmulero — CHANGES_REQUESTED** [review]
Great @migruiz4! only a little suggestion, nothing important

File: bitnami/nginx/templates/NOTES.txt (hunk 1 above)
Comment (migruiz4) [reply bitnami/nginx/templates/NOTES.txt:9]: Updated VIB runtime parameters and NOTES.txt

**Review by fmulero — DISMISSED** [review]
LGTM!

**Review by fmulero — DISMISSED** [review]

**Pull request merged**